modules/scf/chart.sh

```diff
@@ -81,6 +81,8 @@ else
 fi
 
 if  [ "${SCF_OPERATOR}" == "true" ]; then
```

Comment:
I think this makes it work when consuming from s3.kubecf-bundle, but I think it will fail when consuming from s3.kubecf, as it doesn't have those files. Maybe an "if file exists, untar?"

Reply:
yes, what would be the better approach: 
- check for the existence of packaged files in the chart tarball and only extract them if they exist.
- check for the filename of the downloaded chart and handle in dependently